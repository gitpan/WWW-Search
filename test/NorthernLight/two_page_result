http://www.northernlight.com/nlquery.fcg?ns=025&qr=%2BMadonna+%2Bprofile+%2Bfeatures&si=&cb=&cc=&
http://www.northernlight.com/nlquery.fcg?ho=blizzard&po=5030&qr=%2BMadonna+%2Bprofile+%2Bfeatures&si=&&cb=&cc=&ns=025&nth=011ns=025&qr=%2BMadonna+%2Bprofile+%2Bfeatures&si=&cb=&cc=&
