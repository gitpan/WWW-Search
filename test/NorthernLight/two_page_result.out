http://www.crabtree-pub.com/series/wiprofile/6_new_wip.html
http://www.interlog.com/~funga/1996/oscrea.htm
http://www.musiclinks.co.uk/chilln/kwaku/apr/tidbits.htm
http://www.twbooks.co.uk/crimedigests/april961.html
http://www.br.cc.va.us/vcca/good.html
http://geowww.geo.tcu.edu/faculty/geosoftware2.html
http://www.fundsinteractive.com/21st/ch23.html
http://www.eonline.com/Hot/Features/Madonna/index.html
http://www.dmadance.com/inside/dmaprofile.html
http://www.lifeintl.com/celeb.html
http://www.maltergalleries.com/m29.html
http://www.catawba.k12.nc.us/books/pblue10.txt
http://www.eecs.harvard.edu/~zhwang/Madonna/Mnws9707.html
http://www.singers.com/nylons.html
http://www.picosys.com/corpprof.htm
http://cansingles.net/profiles/cs1194.htm
http://www2.rocktropolis.com/allstar/database/news/9803/16/story5.asp
http://www.acs.ryerson.ca/rrj/print/hitchcock.html
http://secure.northernlight.com/cgi-bin/pdserv?cbrecid=BG19980429120007497&cb=0#doc
http://www.lifeintl.com/celeb.html
http://www.maltergalleries.com/m29.html
http://www.catawba.k12.nc.us/books/pblue10.txt
http://www.eecs.harvard.edu/~zhwang/Madonna/Mnws9707.html
http://www.singers.com/nylons.html
http://www.picosys.com/corpprof.htm
http://cansingles.net/profiles/cs1194.htm
http://www2.rocktropolis.com/allstar/database/news/9803/16/story5.asp
http://www.acs.ryerson.ca/rrj/print/hitchcock.html
http://secure.northernlight.com/cgi-bin/pdserv?cbrecid=BG19980429120007497&cb=0#doc
http://www.computerwire.com/bulletins/cstb/21ca_18a.htm
http://www.york.ac.uk/inst/ctipsych/web/CTI/Index.html
http://me210abc.stanford.edu/94-95/projects/Pfizer/Spring/1.html
http://www.showtimearabia.com/dailies/26.htm
http://www.showtimearabia.com/dailies/12.htm
http://www.rockontv.com/mon.html
http://secure.northernlight.com/cgi-bin/pdserv?cbrecid=BM19980519020046702&cb=0#doc
http://secure.northernlight.com/cgi-bin/pdserv?cbrecid=BM19980805010063162&cb=0#doc
